### DISCUSSION TITLE
Maya 2020 support plugin features

### DISCUSSION THREAD

**Discussion by ActiveXSturcture** [Autodesk/maya-usd#1666] [Q&A]
How many features that Maya 2022 shows will compatible in Maya2020,what kind of USD version and Maya devkit version should i use to enable as much features as possible?

**Comment by wtelford** [comment]
Maya 2022 ships with UFE 2.0 (Universal Front End), while 2019 and 2020 shipped with UFE 1.0
Many of the proxyshape workflows require UFE 2.0. Here is a relatively comprehensive list of features that require UFE 2.0 (Maya 2022.x).
- re-parenting
- undo/redo
- duplicate
- rename
- reorder
- outliner visualization of mixed-data model, data type icons, context menus, usd display filters, selection framing
- attribute editor
- some VP2 performance enhancements
- advanced manipulation of arbitrary transform stacks
- viewport selection framing
- viewport purpose filtering
- mixed data model compute
- advance serialization

MayaUSD's translation framework (import/export workflows) and Layer Editor do not leverage UFE, so should work in pre-Maya 2022 builds.

Out latest, MayaUSD 0.13.0, leverages USD 21.08
